sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/SqlScriptingParserSuite.scala

```diff
@@ -269,6 +269,94 @@ class SqlScriptingParserSuite extends SparkFunSuite with SQLHelper {
     assert(exception.origin.line.contains(3))
   }
 
+  test("compound: forbidden label - system") {
```

Comment:
`system` looks ok? `system.var_name` is not a valid name of session variables anyway.